reconcile/utils/terrascript_aws_client.py

```diff
@@ -1177,6 +1179,19 @@ def populate_vpc_peerings(self, desired_state):
                     tf_resource = aws_route(route_identifier, **values)
                     self.add_resource(infra_account_name, tf_resource)
 
+    def populate_vpc_requests(self, vpc_requests: list[VPCRequest]) -> None:
```

Comment:
Lets use `Iterable[VPCRequest]` to stay flexible, see https://service.pages.redhat.com/dev-guidelines/docs/appsre/python-style-guide/#abstract-collection-types

Reply:
Solved.